draft-ietf-quic-version-negotiation.md

```diff
@@ -210,6 +189,54 @@ balancers to ensure that packets for a given connection are routed to the same
 server.
 
 
+# Server Deployments of QUIC {#server-fleet}
+
+While this document mainly discusses a single QUIC server, it is common for
+deployments of QUIC servers to include a fleet of multiple server instances. We
+therefore define the following terms:
+
+Supported Versions:
+
+: This is the set of versions supported by a given individual server instance.
+
+Fully-Deployed Versions:
+
+: This is the set of QUIC versions that is supported by every single QUIC
+server instance in this deployment.
+
+Partially-Deployed Versions:
+
+: This is the set of versions supported by a given individual server instance,
+but which aren't supported by every single QUIC server instance in this
+deployment.
+
+If a deployment only contains a single server instance, then the Supported
+Versions is equal to the Fully-Deployed Versions, and the Partially-Deployed
+Versions is the empty set.
+
+Conversely, if a deployment contains multiple server instances, it is possible
+for software updates to not happen at the exact same time on all server
+instances. In this scenario, some instances will have a different Supported
+Versions from other instances in the same fleet. When that happens, a client
+might receive a Version Negotiation packet from a server instance that doesn't
+support some Partially-Supported Versions; in that case, those
+Partially-Supported Versions won't be listed in the Version Negotiation packet.
+When the client reacts to that Version Negotiation packet, it might create a
+new QUIC connection which could be routed to a different server instance, which
+has a different set of Supported Versions. The version downgrade prevention
+mechanism described in {{downgrade}} is robust to this scenario and will not
+falsely detect a downgrade attack.
+
+If a server operator is progressively deploying a new QUIC version throughout
```

Comment:
Based on the above, I would remove the condition here: just describe how a new version is deployed.